### PR TITLE
Check if hooks folder exists before using it

### PR DESCRIPTION
**Pull request by rbleuse**
Fixes https://github.com/JLLeitschuh/ktlint-gradle/issues/557

### CODE REVIEW

File: plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/GitHook.kt [hunk 1]
```diff
@@ -169,6 +169,12 @@ open class KtlintInstallGitHookTask @Inject constructor(
         }
 
         logger.info(".git directory path: ${repo.directory}")
+        val gitHookDirectory = repo.directory.resolve("hooks")
+        if (!gitHookDirectory.exists()) {
+            logger.info("hooks directory doesn't exist, creating one")
```
Comment (JLLeitschuh) [inline plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/GitHook.kt:174]:
```suggestion
            logger.info("git hooks directory doesn't exist, creating one")
```

**Review by JLLeitschuh — CHANGES_REQUESTED** [review]
This change looks good in general. Please add release notes. 🙂

**Comment by rbleuse** [comment]
> This change looks good in general. Please add release notes. 🙂

@JLLeitschuh thanks, done, please check 🙂

**Review by JLLeitschuh — APPROVED** [review]
LGTM. I'm going to let the CI jobs run before merging

**Pull request merged**